### ISSUE TITLE
Title and/or desc in SVG icons for better accessibility

### ISSUE THREAD

**Issue by samikeijonen** [WordPress/twentyseventeen#394]
All SVG icons are decorative in a theme. `twentyseventeen_get_svg` function generates this markup by default.

```
<svg class="icon icon-next" aria-hidden="true" role="img">
   <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-next"></use>
</svg>
```

`aria-hidden="true"` is added by default because all used icons are decorative. But it's possible that child themes want to use meaningful, non decorative SVG icon. Twenty17 should provide example how to do it as accessible as possible.

In `twentyseventeen_get_svg` function you can already set title and desc parameters. But I'm not sure if we have it correct. At the moment markup looks like this:

```
<svg class="icon icon-next" aria-hidden="true" aria-labelledby="title desc" role="img">
   <title>Title you set</title>
   <desc>Desc you set</desc>
   <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-next"></use>
</svg>
```

Should `aria-labelledby` use unique IDs like this:

```
<svg class="icon icon-next" aria-hidden="true" aria-labelledby="title-id desc-id" role="img">
   <title id="title-id">Title you set</title>
   <desc id="desc-id">Desc you set</desc>
   <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-next"></use>
</svg>
```

We could use `uniqid` function for this in similar way than in `searchform.php`.

Any feedback is welcome.

**Comment by davidakennedy** [comment]
After chatting with the accessibility team, it was determined that further enhancements aren't needed. See: https://wordpress.slack.com/archives/accessibility/p1476728678002426
